commons-packet/commons-packet-manager/src/main/java/io/mosip/commons/packet/util/PacketValidator.java

```diff
@@ -100,13 +100,16 @@ private boolean validateSchema(String id, String source, String process) throws
                     String jsonString = new String(bytearray);
                     LinkedHashMap<String, Object> currentIdMap = (LinkedHashMap<String, Object>) mapper
                             .readValue(jsonString, LinkedHashMap.class).get(IDENTITY);
+                    if (currentIdMap.get(IDSCHEMA_VERSION) != null)
+                        currentIdMap.put(IDSCHEMA_VERSION, Double.valueOf(currentIdMap.get(IDSCHEMA_VERSION).toString()));
```

Comment:
Why are we doing this convertion, as per schema, id schema version is suppose to be string or double ?

Reply:
The idschema is a double now. But i am converting string to double in code to support backward compatibility. Earlier it was string.